src/librustc_target/Cargo.toml

```diff
@@ -10,6 +10,7 @@ crate-type = ["dylib"]
 
 [dependencies]
 bitflags = "1.0"
+syntax_pos = { path = "../libsyntax_pos" }
```

Comment:
I don't see this dependency actually being used -- but maybe I'm missing something?

Reply:
`rustc_target` shouldn't have any dependencies on the compiler, so hopefully this is unused.

Reply:
It is used for custom address space names.

Oh, it was. I forgot I had removed it.